README.md

```diff
@@ -0,0 +1,173 @@
+
+### Key Features <a name="key-features"></a>
+
+- Add a new game, a music album and a book
+- Add extra details such genre, label and authors
+- All the data is persisted in JSON files
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A text editor preferably [Visual Studio code](https://code.visualstudio.com/)
+- Latest version of [Ruby](https://www.ruby-lang.org/en/downloads/) installed
+- [PostgreSQL Server](https://www.postgresql.org/download/)
+
+
+### Setup
+
+Clone this [repository]()https://github.com/nunegabriel/Catalog-of-my-things.git to your desired folder:
+
+```sh
+  cd my-folder
+  https://github.com/nunegabriel/Catalog-of-my-things.git
+  cd catalog_of_my_things
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command in your terminal:
+
+```sh
+  ruby src/main.rb
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Nowembabazi Nickson**
+
+- GitHub: [@nowembabazi](https://github.com/Nowembabazi)
+- Twitter: [@NowembabaziN](https://twitter.com/NowembabaziN)
+- LinkedIn: [NOWEMBABAZI-NICKSON](http://linkedin.com/in/nowembabazi-nickson/)
+
+👤 **your name**
+
+- GitHub: [@nowembabazi](https://github.com/Nowembabazi)
+- Twitter: [@NowembabaziN](https://twitter.com/NowembabaziN)
+- LinkedIn: [NOWEMBABAZI-NICKSON](http://linkedin.com/in/nowembabazi-nickson/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] A graphical user interface
+- [ ] Better information validation
+- [ ] A backend to store the data
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please leave a star 😁
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank the Microverse staff for all the efforts they put in define the requirements of this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **Can I reuse this code?**
+
+  - Yes, feel free to fork it
+
+- **Do I need knowledge of Ruby to use this project?**
+
+  - No you do not.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
### The MIT license link is currently broken, kindly fix it to make your README more professional 👍🏽